prometheus_client/exposition.py

```diff
@@ -23,10 +24,37 @@
     from urllib.parse import quote_plus
 
 
-CONTENT_TYPE_LATEST = 'text/plain; version=0.0.4; charset=utf-8'
+CONTENT_TYPE_LATEST = str("text/plain; version=0.0.4; charset=utf-8")
```

Comment:
> Why the str?
Because of the future import for unicode_literals, WSGI needs a str object. It does not make any difference for the BasicHTTPServer.
> We're using single quotes in this file.
Done